can-globals-proto.js

```diff
@@ -7,6 +7,12 @@ function dispatch(key, value) {
 	var handlers = this.eventHandlers[key];
 	if (handlers) {
 		var handlersCopy = handlers.slice();
+		if (typeof value === 'function') {
+			value = value();
+		}
+		if (value === undefined) {
+			value = this.properties[key].default;
```

Comment:
@andrejewski so you're thinking something like this?

Reply:
@imaustink Yep. But unless I'm misunderstanding the code, `default` can also be a function so that would need called as well to return a value.